Sources/SwiftSyntaxBuilderGeneration/String+Extensions.swift

```diff
@@ -0,0 +1,17 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+extension StringProtocol {
+    var firstLowercased: String { prefix(1).lowercased() + dropFirst() }
```

Comment:
How does this behave on an empty string? Also maybe `withFirstCharacterLowercased` would be clearer.